lib/EResourceType/tests/interactor.js

```diff
@@ -4,5 +4,5 @@ import {
 } from '@bigtest/interactor';
 
 export default @interactor class EresourceTypeInteractor {
-  type = text('[data-test-eresource-type]');
+  publicationType = text('[data-test-eresource-type]');
```

Comment:
Could we change `data-test-eresource-type` to `data-test-eresource-publication-type`.